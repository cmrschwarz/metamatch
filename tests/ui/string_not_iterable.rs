use metamatch::unquote;

fn main() {
    unquote! {
        "asdf".map(|x|x)
    }
}
