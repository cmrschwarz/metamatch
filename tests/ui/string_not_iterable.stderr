error: first argument to `map()` must be iterable, `str` is not
 --> tests/ui/string_not_iterable.rs:5:16
  |
5 |         "asdf".map(|x|x)
  |                ^^^
